backend/apps/owasp/management/commands/owasp_aggregate_contributions.py

```
@@ -0,0 +1,259 @@
+"""Management command to aggregate contributions for chapters and projects."""
+
+from datetime import datetime, timedelta
+from typing import Dict
```

Ruff (0.14.5)

`typing.Dict` is deprecated, use `dict` instead

(UP035)